package.json

```diff
@@ -15,8 +15,8 @@
   },
   "homepage": "https://github.com/recurly/react-recurly#readme",
   "peerDependencies": {
-    "react": "^16.8.0-0",
-    "react-dom": "^16.8.0-0"
+    "react": ">= 16.8.0-0",
```

Comment:
Why include these changes?

Reply:
I did this for better peer dependency resolution when the parent dep is using react 17, helping to alleviate some brittleness that can cause.